src/main/java/frc/robot/Commands/DropPlacer.java

```diff
@@ -0,0 +1,27 @@
+package frc.robot.Commands;
```

Comment:
why do you need to create a class for it? you can just use `MoveGenericSubsystem` and override the `isFinished()` method

Reply:
And init? I think I deserves it own class